pkg/agent/cniserver/sriov.go

```diff
@@ -0,0 +1,130 @@
+package cniserver
+
+import (
+	"fmt"
+	"context"
+	"net"
+	"os"
+	"path"
+	"time"
+	"google.golang.org/grpc"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"antrea.io/antrea/pkg/agent/util"
+	podresourcesv1alpha1 "k8s.io/kubelet/pkg/apis/podresources/v1alpha1"
+	sriovcniutils "github.com/k8snetworkplumbingwg/sriov-cni/pkg/utils"
```

Comment:
I wonder if this package could be used to replace `"github.com/Mellanox/sriovnet"` in a future PR?

Reply:
Yes